src/Client.php

```diff
@@ -2,8 +2,10 @@
 
 namespace Upr\Client;
 
-use GuzzleHttp\Client  as GuzzleClient;
+use GuzzleHttp\Client as GuzzleClient;
 use Symfony\Component\Cache\Adapter\FilesystemAdapter;
+//use Psr\SimpleCache\CacheInterface;
+use Symfony\Contracts\Cache\CacheInterface;
```

Comment:
Please use the psr interface instead. It is universal, and more implementations exist besides symfony.